src/pages/blog/20221202-meetcommerce.md

```diff
@@ -0,0 +1,60 @@
+---
+layout: '@/layouts/post.astro'
+date: 2022-12-02
+title: Meet you soon in Madrid!
+description: Meet Commerce Spain is taking place on Tuesday 13th December, in Madrid, Spain, an Mage-OS will meet you there!
```

Comment:
This description will probably be too long and will get cut off on the social embed image. Please shorten it to 1/2-2/3 the length. See: https://twitter.com/mage_os/status/1595605602352652288

Maybe
> Mage-OS will be at Meet Commerce Spain on December 13th

Otherwise this looks good!